tempesta_fw/t/unit/test_tfw_str.c

```diff
@@ -29,11 +29,11 @@ TEST(tfw_strcpy, zero_src)
 {
 	TfwStr s1 = {
 		.len = 0,
-		.ptr = NULL
+		.data = NULL
 	};
 	TfwStr s2 = {
 		.len = 3,
-		.ptr = "abc"
+		.data = "abc"
```

Comment:
There are macros `DEFINE_TFW_STR()` and `TFW_STR_FROM()` that can be used for this purpose.